### DISCUSSION TITLE
Order of Area cards, Image Preview for cards?

### DISCUSSION THREAD

**Discussion by diothar** [dwainscheeren/dwains-lovelace-dashboard#412] [Dwains Dashboard V3 Alpha]
How is the order of area cards determined? 

I would love an option to have a blank area card be on the homepage to the right of all the area cards, so I can have all my favorites there.  You allow us to add cards there, but right now it associates with the first room.  Maybe a blank room or something? 

Area cards in lovelace let you use the live camera feed as their image preview.  It would be nice to have that on the homepage as well.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
